skyline-vscode/react-ui/src/utils/parsers.js

```diff
@@ -0,0 +1,97 @@
+import Ajv from "ajv";
+import {
+  gpuPropertyList,
+  deploymentScatterGraphColorSize,
+} from "../data/properties";
+import { cloudProviderSchema } from "../schema/CloudProvidersSchema";
+
+const ajv = new Ajv({ allErrors: true }); // to report all validation errors (rather than failing on the first errors)
+const validate = ajv.compile(cloudProviderSchema);
+
+class ResponseBuffer {
+  constructor() {
+    this.instanceId = 0;
+    this.instanceArray = [];
+    this.cloudProviders = {};
+    this.errors = [];
+  }
+}
+export const loadJsonFiles = async (habitatData, additionalProviders) => {
+  const buffer = new ResponseBuffer();
+  let urlList = [
+    "https://deepview-explorer-public.s3.amazonaws.com/vscode-cloud-providers/providers.json",
```

Comment:
This should be stored as a constant in properties.js

Reply:
moved variable to properties.js